docs/how-to/join-a-network.md

```diff
@@ -101,8 +101,9 @@ Here’s how to upgrade in place, with minimum impact:
 * If you are running miner(s), check if your miner is safe to shut down and restart: `lotus-miner proving info`. If any deadline shows a block height in the past, do not restart. See examples below.
 * Otherwise, you can safely restart your daemon and miner(s). This will replace and update the binary, and should preserve your config, wallets, and anything else on disk.
 
-In the following example, Deadline Open is 454 which is earlier than Current Epoch of 500. This miner should **not** be shut down or restarted.
-```
+In the following example, `Deadline Sectors` is 0 which means no sector is currently being challenged. And this dealine is going to closed in 7 minutues, as indicated by `Dealdline Close`. This means the miner can be safely restarted in the next 7 minutes, before the next deadline reaches.
```

Comment:
```suggestion
In the following example, `Deadline Sectors` is 0 which means no sectors are currently being challenged in this deadline. This deadline closes in 7 minutes, as indicated by `Deadline Close`. This means the miner can be safely restarted in the next 7 minutes, before the next deadline reaches.
```

Question for @travisperson @arajasek is it risky to restart with 7 minutes remaining?